modules/navier_stokes/src/fvkernels/INSFVMixingLengthScalarDiffusion.C

```diff
@@ -67,16 +58,18 @@ INSFVMixingLengthScalarDiffusion::computeQpResidual()
 #ifdef MOOSE_GLOBAL_AD_INDEXING
   constexpr Real offset = 1e-15; // prevents explosion of sqrt(x) derivative to infinity
 
-  const auto & grad_u = _u_var->adGradSln(*_face_info);
+  auto face = makeCDFace(*_face_info);
```

Comment:
so when we make an argument like this, it defaults to no skewness correction right?
but what if the variable has skewness-correction on, does that bring it back?

Reply:
No it doesn't. `adGradSln` takes a `correct_skewness` argument which defaults to `false`, so the change here preserves previous behavior